content/SCALE/SCALETutorials/Storage/Disks/SLOGOverprovisionSCALE.md

```diff
@@ -16,27 +16,135 @@ Over-provisioning an SSD distributes the total number of writes and erases acros
+    |       critical | <null>                                         |
+    |     difference | <null>                                         |
+    |  informational | <null>                                         |
+    |          model | SAMSUNG_MZ7KM240HAGR-0E005                     |
+    |   rotationrate | <null>                                         |
+    |           type | SSD                                            |
+    |       zfs_guid | 3102327624417314756                            |
+    |            bus | ATA                                            |
+    |        devname | sds                                            |
+    |      enclosure | <dict>                                         |
+    | supports_smart | <null>                                         |
+    |           pool | <null>                                         |
+    +----------------+------------------------------------------------+
+    ```
+
+   Confirm the target disk reports the updated size.
+   Check if the reported **name** has changed.
+   If so, record the new name.
+
+7. Create a new pool or locate an existing pool on the **Storage Dashboard** and (re)assign the SLOG device to bring it online.
+
+   {{< expand "To create a new pool with a SLOG device" "v" >}}
+
+   a. Go to **Storage** in the TrueNAS SCALE UI and click **Create Pool** to open the **Pool Creation Wizard**.
+
+   b. Enter a name for the Pool and create at least one **Data** storage VDEV that fits your use case requirements.
+
+   c. Click **Log** and select a **Layout**.
+
+   d. Click **Manual Disk Selection** to open the **Manual Selection** dialog, click **Add** to add a VDEV, and search by name or serial number to locate the resized SLOG disk.
+
+   e. Drag and drop the disk to the VDEV and click **Save Selection**.
+
+   f. Add any other optional VDEVs as needed for your storage and performance requirements.
+
+   e. Click **Create Pool**.
+
+   See [Create Pool]({{< relref "/SCALE/scaletutorials/storage/createpoolwizard.md" >}}) for more information on using the **Pool Creation Wizard**.
+
+   {{< /expand >}}
+
+   {{< expand "To (re)assign the SLOG device to an existing pool" "v" >}}
+
+   a. Go to **Storage** in the TrueNAS SCALE UI, locate the pool that you want to assign the SLOG device to and click **Manage Devices** on the **Topology** widget to open the **Devices** screen.
+
+   b. Click **Add VDEV** to open the **Add Vdevs to Pool** screen.
+
+   c. Click **Log** and select a **Layout**.
+
+   d. Click **Manual Disk Selection** to open the **Manual Selection** dialog, click **Add** to add a VDEV, and search by name or serial number to locate the resized SLOG disk.
+
+   e. Drag and drop the disk to the VDEV and click **Save Selection**.
+
+   f. Go to **Review** and click **Update Pool**
+
+       See [Managing Pools]({{< relref "/SCALE/scaletutorials/storage/managepoolsscale.md" >}}) for more information on using **Add Vdevs to Pool**.
```

Comment:
This is not rendering properly. It shows up as a code block of some sort. Not sure why it is doing it unless it is the extra spaces in line 148.

Reply:
That line was just indented too far. Probably showed up when I added the expand and I didn't recheck.